src/components/pages/txs/latest-l1-l2-transactions-table.tsx

```diff
@@ -0,0 +1,75 @@
+import Link from "next/link";
+import { formatTimestamp } from "@/lib/utils";
+import { Card, CardContent } from "@/components/ui/card";
+import { L1L2Transaction } from "@/lib/types";
+import { SquareArrowOutUpRight } from "lucide-react";
+import {
+  Table,
+  TableCaption,
+  TableHeader,
+  TableRow,
+  TableHead,
+  TableBody,
+  TableCell,
+} from "@/components/ui/table";
+import { l1Chain } from "@/lib/chains";
+
+const LatestL1L2TransactionsTable = ({
+  transactions,
+}: {
+  transactions: L1L2Transaction[];
+}) => (
+  <Card>
+    <CardContent className="p-4">
+      <Table className="w-full table-auto">
+        <TableCaption>
+          List of latest L1 - L2 transactions
+        </TableCaption>
+        <TableHeader>
+          <TableRow>
+            <TableHead className="max-w-[10rem]">Block No</TableHead>
+            <TableHead className="max-w-[10rem]">L2 Tx Hash</TableHead>
+            <TableHead className="max-w-[10rem]">Timestamp</TableHead>
+            <TableHead className="max-w-[10rem]">L1 Tx Hash</TableHead>
+            <TableHead className="max-w-[10rem]">L1 Tx Origin</TableHead>
+            <TableHead className="max-w-[8rem]">Gas Limit</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {transactions.map((transaction) => (
+            <TableRow key={transaction.l2Hash}>
+              <TableCell className="max-w-[10rem] truncate">
+                <Link href={`https://explorer.l1.com/block/${transaction.l1BlockNumber}`}>
```

Comment:
Use the block explorer from the chain config in l1Chain defined here:
https://github.com/walnuthq/op-scan/blob/main/src/lib/chains.ts#L7

Use the proper styling as seen on the homepage: `text-primary hover:brightness-150`